第12回演習課題

Hikaru-Sasa opened the pull request:
students テーブルに備考と削除フラグを追加

Hikaru-Sasa committed:
students テーブルに備考と削除フラグを追加

KeitaUenishi commented on src/main/java/raisetech/StudentManagement/Controller/converter/StudentConverter.java:
```diff
@@ -0,0 +1,31 @@
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+import raisetech.StudentManagement.data.StudentsCourses;
+import raisetech.StudentManagement.data.student;
```
クラス名が先頭小文字で定義されていますかね？
`student` → `Student` に修正しておきましょう

KeitaUenishi commented on src/main/java/raisetech/StudentManagement/domain/StudentDtail.java:
```diff
@@ -0,0 +1,18 @@
+
+@Getter
+@Setter
+public class StudentDtail {
```
`StudentDetail`のタイポですね！
修正しておきましょう

Hikaru-Sasa committed:
students テーブルに備考と削除フラグを追加

Hikaru-Sasa committed:
第15回演習課題
新規受講生情報を登録する処理を実装しましょう
上記の実装ができて時間がある方は
コース情報も一緒に登録できるように実装しましょう
その際に、登録できるコースは１つのみで構いません

Hikaru-Sasa committed:
第16回演習課題
受講生更新のHTMLを作成しましょう
（受講生登録を参考にしましょう）
受講生一覧の表で名前をクリックすると、その受講生のID情報に
もとづいたデータを表示できるようにしましょう

Hikaru-Sasa committed:
第17回演習課題
更新作業の見直しをしましょう
受講生更新のページに、キャンセルができるようなチェックボックスを作成しましょう
受講生一覧の表に、キャンセルされたデータが表示されないようにしましょう

Hikaru-Sasa committed:
第21回演習課題
検索の部分を中心にリファクタリングを行いましょう
動作確認をしておきましょう
時間がある方は、他の部分もリファクタリングを行いましょう

Hikaru-Sasa committed:
第21回演習課題
検索の部分を中心にリファクタリングを行いましょう
動作確認をしておきましょう
時間がある方は、他の部分もリファクタリングを行いましょ

Hikaru-Sasa committed:
第22回演習課題
Student Repositoryに書かれているコードを
全てXML形式で記述しましょう
入力チェックを試してみましょう
